.changeset/sweet-crews-beg.md

```diff
@@ -1,6 +1,4 @@
 ---
-"@sap-ux-private/control-property-editor-common": patch
-"@sap-ux/control-property-editor": patch
 "@sap-ux-private/preview-middleware-client": patch,
```

Comment:
Whats happening here with the changeset files? Earlier builds were failing because of a bad changeset and oddly similar to the one that was causing a problem and needed this fix https://github.com/SAP/open-ux-tools/commit/cc848155a47834881fd07efff2042bff272451a8.

Reply:
updated manually.